opencga-catalog/src/main/java/org/opencb/opencga/catalog/auth/authentication/AzureADAuthenticationManager.java

```diff
@@ -58,22 +58,17 @@
 
 public class AzureADAuthenticationManager extends AuthenticationManager {
 
-    private String originId;
```

Comment:
Why have you changed the order of those fields? If you are using a plugin that is doing that kind of changes, it would be desirable to disable it.